src/views/Entry.vue

```diff
@@ -144,6 +147,12 @@ export default {
         return
       }
       return this.config.dataDefinition
+    },
+    mostlyDisabled: function () {
+      if (!this.$route.params.id) {
```

Comment:
Why not just enable it? If you leave it blank it will generate one. If you specify it, that will be used. If there is a conflict, the working branch version #12 will display an error.